code/game/machinery/pipe/construction.dm

```diff
@@ -160,13 +160,13 @@ Buildable meters
 	var/list/potentially_conflicting_machines = list()
 	// Work out which machines we would potentially conflict with
 	for(var/obj/machinery/atmospherics/machine in loc)
-		// skip checks if we don't overlap layers, either by being on the same layer or by something being on all layers
-		if(machine.piping_layer != piping_layer && !((machine.pipe_flags | flags) & PIPING_ALL_LAYER))
-			continue
 		// Only one dense/requires density object per tile, eg connectors/cryo/heater/coolers.
-		if((machine.pipe_flags & flags & PIPING_ONE_PER_TURF))
+		if((machine.pipe_flags & flags) & PIPING_ONE_PER_TURF)
```

Comment:
this isn't really needed, A & B & C is about the same as B & C & A, since if a bit isn't in any one of the flags it's not gonna be in the output